### ISSUE TITLE
Bug: Unable to open right panel for some customer on new Gmail  version

### ISSUE THREAD

**Issue by fstn** [InboxSDK/InboxSDK#1070]
When opening the right Panel:

```
    index.8b6ab181.js:55 Error logged: Error: Failed to find element with selector: table.Bs > tr
    at c (index.8b6ab181.js:145:13093)
    at l8._startMonitoringPreviewPaneForThread (index.8b6ab181.js:226:4867)
    at l8._setupContentAndSidebarView (index.8b6ab181.js:226:4481)
    at index.8b6ab181.js:226:3793
    at z.call (index.8b6ab181.js:112:3465)
    at MutationObserver.x (index.8b6ab181.js:112:3706)
 ```

When opening a sent email:

<img width="1334" alt="image" src="https://github.com/InboxSDK/InboxSDK/assets/2936857/f1426bf7-36dc-4d08-8be5-3cd94b0d017c">

"@inboxsdk/core": "^2.0.1", and tried last 2.1.13 
Macbook
Chrome Version 119.0.6045.159 (Official Build) (arm64)

Regard

**Comment by wegry** [comment]
Hey @fstn, thanks the report. 

#1073, released as part of `@inboxsdk/core@2.1.15`, might help with this issue.

**Comment by wegry** [comment]
Closing as dupe of #1072.
